music_assistant/providers/builtin_player/__init__.py

```diff
@@ -0,0 +1,396 @@
+"""Built-in Player Provider for playing to the Music Assistant Web Interface."""
```

Comment:
We could consider making the description a bit for universal as this is potentially not only suitable for the MA web interface but it creates a HTTP based generic player, which can be utilized by/in the MA frontend but potentially also be consumed by the user as an URL or even the HA mediabrowser player.

Reply:
Also, I'm working on a "local audio output" plugin which could easily benefit from this as well. A builtin player which can work out of the box (without the need for any 3rd party protocols)